app/src/main/java/com/kozyrev/myfirstapplication/screens/MainActivity.java

```diff
@@ -235,6 +235,7 @@ private String readStream(InputStream in) {
                 } catch (IOException e) {
                     e.printStackTrace();
 
+                    Log.i("abc", "first log");
```

Comment:
please don't leave log statements without appropriate log tag

Reply:
I think it's just fine